src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Unix.cs

```diff
@@ -72,9 +72,5 @@ internal static void ThrowIfRemoteMachine(string machineName)
                 throw new PlatformNotSupportedException(SR.RemoteMachinesNotSupported);
             }
         }
-        public static IntPtr GetMainWindowHandle(int processId)
```

Comment:
as pointed by @buyaa-n in https://github.com/dotnet/runtime/issues/57090#issue-964393160 it turned out to be dead code